### ISSUE TITLE
Feature Request: Log to SDCard

### ISSUE THREAD

**Issue by torabaito** [cool4uma/UART_Terminal#9]
is it possible to stream RX to a log file on the SDCard?

**Comment by cool4uma** [comment]
Unfortunately, there is no such possibility in the current implementation. Recently, I myself had such a need, I plan to implement it in the near future.

**Comment by maldomen** [comment]
Hello, I'm in need of an uart logger for a recent project and a handheld like a flipper zero would come very handy. Is there any plans on this? Is there any alternative app?

**Comment by sjn** [comment]
Yes, this would be super useful! Either this, or to save the terminal text to a file.
